client/components/section-nav/tabs.jsx

```diff
@@ -139,7 +140,7 @@ var NavTabs = React.createClass( {
 				return;
 			}
 
-			navGroupWidth = this.refs.navGroup.getDOMNode().offsetWidth;
+			navGroupWidth = this.refs.navGroup.offsetWidth;
```

Comment:
>(Or maybe not since one is a React component and the other is a DOM component?)

This is correct. When the ref is applied to a DOM component (in this case, a `div`), the ref returns the DOM node directly, whereas in the case where it is a React element (in the above case, a `<SectionTab />`), we still must use `ReactDom.findDOMNode` if we want access to its rendered node.